package.json

```diff
@@ -39,6 +39,7 @@
     "dcos-dygraphs": "1.1.0-beta.3",
```

Comment:
Hey @tnguyen0 did you follow the docs when adding this packages? https://github.com/dcos/dcos-ui/blob/master/CONTRIBUTING.md#adding-npm-package-dependencies

Just want to make sure because I had issues the first time adding a new package to the.